### ISSUE TITLE
UX Suggestion: When at the top and swipe upwards, go into search

### ISSUE THREAD

**Issue by TobiasDev** [authpass/authpass#190]
Since phones are just getting bigger and bigger I find that I can't reach the "Search" icon without moving my hands or use both (I have small hands). Therefore I would like to suggest that if I'm already at the top of my KeePass list and try to scroll up it goes into search mode and open my keyboard. That way I could easily access search even with one hand.

**Comment by hpoul** [comment]
Interesting idea.. I can relate to the problem.. I'm not sure about the solution though.. scrolling up the top is usually related to refreshing the list? 🤔️

btw. i think going forward i would like to try to discuss ideas (vs specific implementation tasks/bugs) more in the forum: https://forum.authpass.app/c/feature-discussions/7 .. my hope is that more non-technical users would join the discussion 😅️

**Comment by TobiasDev** [comment]
Oh, sorry! I didn't see that you had a forum! Of-course, that is a way better location. 

And I do agree with the "referesh" part, I didn't actually think about that... But now that you say it of-course that is what it usually does and when you use "Web-dav" that is def. something you will want to do. Hmm, maybe doing a bottom-bar like a lot of new apps do (Deezer, Firefox, etc.), but that can be a discussion on the forum! :)

**Comment by hpoul** [comment]
👍️ didn't you get the in-app banner? :-D (probably not displayed for f-droid users though)
anyway, i've now added issue templates to link to the forum :-) https://github.com/authpass/authpass/issues/new/choose

**Comment by hpoul** [comment]
Well, thinking about it.. except moving the search box to the bottom vs the top (like you said, firefox).. we could change the floating action button from just being "create new password" to offer more actions.. 

<img width="295" alt="Screen Shot 2020-10-21 at 22 41 29" src="https://user-images.githubusercontent.com/313066/96785142-ab2e1400-13ee-11eb-93bb-9b1524fc7ae9.png">

when there are multiple files open it already pops open with one action for each file.. we could easily add a 'search' button as the lowest action.. would this improve things? you'd have to tap twice ab the bottom, instead of once in the top :-)

**Comment by TobiasDev** [comment]
Yeah, that would def. improve things. It's really only since I need to "struggle" to reach the top. :D

**Comment by hpoul** [comment]
I've created a poll in the forum 😅️ https://forum.authpass.app/t/password-list-bottom-shortcut-to-enter-search/28

let's see if we can get a few opinions on that :-)
